src/views/AllCharacters/AllCharacters.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { filterCharacters, getCharacters } from '../../services/harry-potter';
+import { useEffect, useState } from 'react';
+import CharacterList from '../../components/CharacterList/CharacterList';
+import Controls from '../../components/Controls/Controls';
+
+export default function AllCharacters() {
+  const [characters, setCharacters] = useState([]);
+  const [house, setHouse] = useState('default');
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await getCharacters();
+      const uniqueData = [...new Set(data)];
```

Comment:
I don't think that this will work to remove duplicates as the items inside of your array are objects. https://dmitripavlutin.com/value-vs-reference-javascript/